pages/community.tsx

```diff
@@ -0,0 +1,397 @@
+import {
+  AspectRatio,
+  Box,
+  Center,
+  Container,
+  DarkMode,
+  Flex,
+  Icon,
+  Image,
+  SimpleGrid,
+} from "@chakra-ui/react";
+import { CommunityCard } from "components/community/Card";
+import { HomepageFooter } from "components/footer/Footer";
+import { HomepageTopNav } from "components/product-pages/common/Topnav";
+import { useTrack } from "hooks/analytics/useTrack";
+import { NextSeo } from "next-seo";
+import { PageId } from "page-id";
+import { FiZap } from "react-icons/fi";
+import { Heading, LinkButton, Text } from "tw-components";
+import { ThirdwebNextPage } from "utils/types";
+
+const TRACKING_CATEGORY = "community";
+
+const SEO = {
+  title: "Join the Best Web3 Community | Learn, Build, Grow",
+  description:
+    "Join the thirdweb community and share your web3 journey alongside 35,000+ other builders. Learn, build, & grow with thirdweb.",
+};
+
+const CommunitySections = [
+  {
+    title1: "Learn",
+    title2: "web3 development",
+    description:
+      "Whether you're interested in web3 or thirdweb, our mission is to cultivate a community of like-minded individuals who share our passion for the future of decentralized technology.",
+
+    cards: [
+      {
+        image: "",
+        link: "/learn",
+        title: "thirdweb learn",
+        description: "",
+      },
+      // {
+      //   image: "",
+      //   link: "/learn",
+      //   title: "Bootcamp",
+      // description: ""
+      // },
+      {
+        image: "",
+        link: "/events",
+        title: "Events",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/events",
+        title: "Office Hours",
+        description: "",
+      },
+    ],
+  },
+  {
+    title1: "Build",
+    title2: "innovative projects",
+    description:
+      "From hackathons to bounties, open source to community projects, there are so many ways to build no matter your level of contribution. ",
+    cards: [
+      {
+        image: "",
+        link: "/events",
+        title: "Hackathons",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://www.notion.so/thirdweb/Paid-Projects-e090e7cefdc14c0f8e8ad6aa47dbdb74?pvs=4",
+        title: "Projects",
+        description: "",
+      },
+      {
+        image: "",
+        link: "https://thirdweb.notion.site/thirdweb-Bug-Bounty-Program-f78d1db776ab4f0e974c9da176fcf706",
+        title: "Bug Bounty",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/oss",
```

Comment:
I think the link changed to /open-source?

Also if we are gonna have more pages I think we should scope them all under /community and have /community/open-source?

Reply:
yeah let's do that- and yes it changed to open-source